From: Harald Dunkel <harri@afaics.de>
Date: Tue, 24 Mar 2015 16:13:43 +0100
Subject: Define_GNU_SOURCE to enable GNU extensions.

The upstream def.h enables _GNU_SOURCE macro, but alas after necessary
includes in the source code. We define _GNU_SOURCE in GNUmakefile and
remove definition in def.h since they both conflict.
---
 GNUmakefile | 1 +
 def.h       | 8 --------
 2 files changed, 1 insertion(+), 8 deletions(-)

Index: mg/GNUmakefile
===================================================================
--- mg.orig/GNUmakefile
+++ mg/GNUmakefile
@@ -34,6 +34,7 @@ CFLAGS+=	-g -Wall
 CPPFLAGS+=	-DREGEX
 CPPFLAGS+=	-D_GNU_SOURCE
 CPPFLAGS+=	$(BSD_CPPFLAGS) -D__dead=__dead2
+CPPFLAGS+=	-D_GNU_SOURCE
 LIBS=		$(CURSES_LIBS) $(BSD_LIBS)
 
 
